app-admin/mokutil/spec

```diff
@@ -0,0 +1,3 @@
+VER=0.6.0
+SRCS="git::commit=tags/$VER::https://github.com/lcp/mokutil.git"
+CHKSUMS="SKIP"
```

Comment:
Please include a `CHKUPDATE=""` parameter here for us to check for updates in the future.

See [here](https://github.com/AOSC-Dev/aosc-findupdate/blob/master/docs/config.zh-CN.md) for details.